server/router.go

```diff
@@ -67,5 +66,5 @@ func RegisterHandlers(s controllers.Server) {
 	adminResourceTypes.PUT("/:resource_type_id", s.UpdateResourceType)
 
 	// Admin plan quota default endpoints.
-	admin.POST("/plan_quota_defaults", s.AddPlanQuotaDefault)
+	admin.POST("plans/quota-defaults", s.AddPlanQuotaDefault)
```

Comment:
I like the change to the new URL format. It should have a leading `/` on the path, though.